src/ui/PageHeading/PageHeading.tsx

```diff
@@ -0,0 +1,77 @@
+import { forwardRef } from "react";
```

Comment:
might be better renaming "Heading" to something about "Header" to make it more UI component like